### ISSUE TITLE
Icons not being displayed with terminator or terminal

### ISSUE THREAD

**Issue by killbotXD** [romkatv/powerlevel10k#639]
I just installed powerlevel10k but after the configuration, none of the icons were shown properly, I tried to look and troubleshoot but couldn't find a solution.

![Screenshot from 2020-04-19 12-37-58](https://user-images.githubusercontent.com/42870378/79681814-c0a56400-823a-11ea-865c-79bbe521d0b7.png)

this is the relevant code that I had in my `~/.zshrc`
I have zsh 5.4.2 on my Ubuntu
and I am also using the recommended font.

```
ZSH_THEME="powerlevel10k/powerlevel10k"
POWERLEVEL10K_MODE="nerdfont-complete"
POWERLEVEL10K_LEFT_PROMPT_ELEMENTS=(os_icon dir newline vcs)
```

**Comment by romkatv** [comment]
Please type `p10k configure`.

**Comment by killbotXD** [comment]
I did try to configure it 6 7 times with a couple of fonts. But none of them worked perfectly. I tried some random combinations of yes and no and I have some icons working but not all.

**Comment by romkatv** [comment]
You don't need to try different fonts. You also don't need to answer wizard's questions incorrectly. Please use [the recommended font](https://github.com/romkatv/powerlevel10k/#meslo-nerd-font-patched-for-powerlevel10k). If by following the instructions you cannot make it work, please ask for help on a forum dedicated to your terminal and/or operating system.

**Comment by killbotXD** [comment]
Okay. I did use the recommended font and also answered the wizard with correct answers. I have the same effect on other terminals also. 

Thanks for the help I will try asking in an Ubuntu forum.

**Comment by sbkpilot1** [comment]
Having a similar problem with icons not showing..

I usually use Powerlevel10K with WSL2 on my other systems and it works terrific, however all those are with Debian and using Windows Terminal configured using TTF Nerd fonts.

But now I am trying it on a new system with Alpine, using the `powerline-ter-v20n.psf.gz` font that I downloaded, I changed `/etc/conf.d/consolefont` to use the font and the font itself works fine but when I ran the powerlevel10k configuration it did not ask me any questions regarding icons at all.. it started the wizard halfway through so now no icons show up. 

The issue here is that I am not able to use TTF fonts, the console uses `.psf.gz`  format.  Do these work with Powerlevel10k?
